packages/knowledge-graph/src/visualization/GraphVisualization.ts

```
@@ -0,0 +1,311 @@
+        return {
+          nodes: graphData.nodes,
+          links: graphData.edges.map((e) => ({
+            source: e.source,
+            target: e.target,
+            ...e,
+          })),
+        };
+
+      case 'graphml':
+        // Generate GraphML XML
+        let graphml = '<?xml version="1.0" encoding="UTF-8"?>\n';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)